apps/admin/integration-testing/Makefile

```diff
@@ -1,15 +1,10 @@
 APP := ..
 export ADMIN_WORKSPACE=/tmp/admin-integration-testing
 
-build-frontend:
+build:
+	# building the frontend builds the backend too
```

Comment:
Is this comment now true because of the `build` change or was it already true?

Reply:
This was kind of true before insofar as the backends are TypeScript dependencies of the frontends, but with the same caveats that it might not actually build _everything_ if there's any non-`tsc` work to be done.